go.mod

```diff
@@ -11,7 +11,7 @@ require (
 	github.com/mr-tron/base58 v1.2.0
 	github.com/multiformats/go-multiaddr v0.3.2
 	github.com/nspcc-dev/hrw v1.0.9
-	github.com/nspcc-dev/neo-go v0.95.3
+	github.com/nspcc-dev/neo-go v0.96.1-pre.0.20210721163421-520133aee2f4
```

Comment:
Is it required to use unreleased version? Can we use v0.96.0?

Reply:
Actually yes, I needed this commit https://github.com/nspcc-dev/neo-go/commit/5bdcd4c241ed7e09811ce98e22e950d398220175 which seems to be in 0.96.0.